app/models/family.rb

```diff
@@ -2,5 +2,8 @@ class Family < ApplicationRecord
   belongs_to :user
   has_many :personal_info, as: :informable, dependent: :destroy
   accepts_nested_attributes_for :personal_info
-  has_and_belongs_to_many :lic
+  has_and_belongs_to_many :life_insurances
```

Comment:
@VaibhaviJoshi-BTC : I think , this should be like family has_and_belongs_to_many :polices instead of lic, because when we move further, we may have more categories, 
so cross check our application flow & reverify the mapping